### PR TITLE
Small refactor of kafka avro functional tests + string data support

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/AvroSchemaOutputValidatorPrinter.scala [hunk 1]
```diff
@@ -48,7 +49,7 @@ object AvroSchemaOutputValidatorPrinter {
       case Schema.Type.ARRAY =>
         s"List[${print(schema.getElementType)}]"
       case Schema.Type.MAP =>
-        s"Map[String, ${print(schema.getValueType)}]"
+        s"Map[String,${print(schema.getValueType)}]"
```
Comment (lciolecki) [inline utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/AvroSchemaOutputValidatorPrinter.scala:52]: Please also remove 'space' from record printing?

**Review by lciolecki — APPROVED** [review]
Nice job!

File: utils/components-testkit/src/main/scala/pl/touk/nussknacker/engine/util/test/TestScenarioRunner.scala [hunk 2]
```diff
@@ -38,4 +38,10 @@ object RunResult {
 
 }
 
-case class RunResult[T](errors: List[NuExceptionInfo[_]], successes: List[T])
+case class RunResult[T](errors: List[NuExceptionInfo[_]], successes: List[T]) {
+
+  def mapSuccesses[U](f: T => U): RunResult[U] = {
```
Comment (lciolecki) [inline utils/components-testkit/src/main/scala/pl/touk/nussknacker/engine/util/test/TestScenarioRunner.scala:43]: I think we don't need here breacets?

File: utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/AvroSchemaOutputValidatorPrinter.scala (hunk 1 above)
Comment (arkadius) [reply utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/AvroSchemaOutputValidatorPrinter.scala:52]: In TypedObjectTypingResult we have version with space:
```
fields.map { case (name, typ) => s"$name: ${typ.display}"}.mkString("{", ", ", "}")
```.

So we are consistent here.

**Pull request merged**